impress-org/givewp
# Donor Dashboard no longer freezes up when attempting to manage Stripe ACH subscriptions

## Description

This PR provides a hotfix solution to an issue which caused the Donor Dashboard to freeze up, when attempting to enter new card information for subscriptions created using the Stripe + Plaid gateway. The PR includes some minor additional improvements to CardControl styling and addresses a React warning about a missing key prop on the SubscriptionRow component.

## Affects

This PR affects frontend rendering logic related to the Subscription Manager view of the Donor Dashboard.

## Visuals

N/A

## Testing Instructions
Test this pull request with: https://github.com/impress-org/give-recurring/pull/1071

1. Setup a Recurring donation using Stripe + Plaid
2. Click "Manage Subscription"
3. Check that you can update the amount, but not the payment method
4. Test subscriptions created using other supported gateways to ensure that everything still works as expected

- [ ] Donor able to update credit card  and donation amount of authorize.net and other stripe payment methods
- [ ] Donor able to update donation amount of GoCardless payment gateway.

## Pre-review Checklist
